### ISSUE TITLE
Incomplete routes are incompatible with destProbReroute

### ISSUE THREAD

**Issue by ishuwen** [eclipse-sumo/sumo#13919]
Add your issue description here.
here is the picture:
<img width="1127" alt="19c07ecf54b1ba306299189de693b61" src="https://github.com/eclipse-sumo/sumo/assets/32593963/7650fc2d-e21d-4d3b-a683-271dcf32a400">

i want test the destProbReroute and closingLaneReroute function, i set the rerouter to close the lane E5 when the simulation time is 5s and choose the other lane E8 as the new destination, but only the vehicles which had departed before the close lane actived could change their destination, the vehicles which depart after the close lane actived is stopped(i use ignore-route-errors, if i dont use it the simulation will generate error)

here is what i set in add file:
```
  <rerouter id="rr_0" edges="E1" pos="86.53,10.29">
        <interval begin="5.00" end="3600.00">
            destProbReroute id="E8" probability="1.00"/>
            <closingLaneReroute id="E5_0" disallow="all"/>
        </interval>
  </rerouter>
```

and if i use destProbReroute only and close lane in the sumo-gui by right button click, it will work normally, what's the difference between closelane rerouter and closelane in sumo-gui? Thanks!

**SUMO-version:**
1.18
**operating system:**
Windows111

**Comment by ishuwen** [comment]
here is what i set in add file:

  <rerouter id="rr_0" edges="E1" pos="86.53,10.29">
        <interval begin="5.00" end="3600.00">
            <destProbReroute id="E8" probability="1.00"/>
            <closingLaneReroute id="E5_0" disallow="all"/>
        </interval>
    </rerouter>

**Comment by m-kro** [comment]
I could not reproduce the problem in a very similar scenario. Could you maybe attach your sample scenario here?

**Comment by ishuwen** [comment]
[reouter.zip](https://github.com/eclipse-sumo/sumo/files/12846475/reouter.zip)
thanks for answering, here is it

**Comment by m-kro** [comment]
It works if you reference the route in the flow:

```
<flow id="f_0" type="td_0" begin="0.00" departLane="random" route="r_0" end="3600.00" vehsPerHour="900.00"/>
```

**Comment by ishuwen** [comment]
it works thanks, but i want know why there is an error if i use "from to", is that bug or i set it wrong? because if i use sumo-gui and right click to close the lane it won't generate error, what's difference between sumo-gui close lane and closelane rerouter?

**Comment by m-kro** [comment]
Yes, for the moment I'd consider it a bug and tagged it accordingly.

**Comment by ishuwen** [comment]
OK，thanks again
